### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lzf971107**
## What is the purpose of the change

Update CONTRIBUTING.md

## Brief changelog

1.CONTRIBUTING.md

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ If your PR contains large changes, e.g. component refactor or new components, pl
 
 ### Code review guidance
 
-Committers will rotate reviewing the code to make sure all the PR will be reviewed timely and by at least one committer before merge. If we aren't doing our job (sometimes we drop things). And as always, we welcome volunteers for code review.
```
Comment (nkorange) [inline CONTRIBUTING.md]: Code review is assigned to Committer, not Committee.

Comment (lzf971107) [reply CONTRIBUTING.md]: Whether the text under the "Now how about try become a committer?" heading is modified correctly

Comment (lzf971107) [reply CONTRIBUTING.md]: If correct I will roll back what you marked

Comment (nkorange) [reply CONTRIBUTING.md]: I think the word 'Committer' is fine. No 'Committee' is needed.

Comment (lzf971107) [reply CONTRIBUTING.md]: Ok, I'll just fix the link failure error.

**Pull request merged**